src/Command.js

```diff
@@ -19,6 +19,47 @@ export default class Command {
     this.concurrency = (!flags || flags.concurrency === undefined) ? DEFAULT_CONCURRENCY : Math.max(1, +flags.concurrency || DEFAULT_CONCURRENCY);
   }
 
+  get name() {
+    // For a class named "FooCommand" this returns "foo".
+    return this.className.replace("Command", "").toLowerCase();
+  }
+
+  get className() {
+    return this.constructor.name;
```

Comment:
Oops, your second comment didn't show up when I was typing my response.

Clashing names would be a problem no matter which way they're specified.

How about errors on:

- Invalid class name
- Name collision

That will make it easier to catch problems in the future.